services/migration/Dockerfile

```diff
@@ -81,10 +81,10 @@ COPY --from=build ${VIRTUAL_ENV} ${VIRTUAL_ENV}
 COPY --chown=scu:scu services/migration/docker services/migration/docker
 
 HEALTHCHECK \
-  --interval=10s \
+  --interval=5s \
```

Comment:
10*3 = 30s vs 5 * 10 = 50s: So you double the frequency  to make it more responsive, but why longer? Is there another problem?

Reply:
it is a small improvement on the starting time until the migration is deemed as healthy. with 10s it means it is never ready below 10s.